tests/integration/plugins/test_signer.py

```diff
@@ -39,6 +39,15 @@ async def verify_signature(records, timestamp, signature):
         await verifier.verify(serialized, signature["signature"], x5u)
 
 
+def fetch_changeset(client, bucket=None, collection=None, **kwargs):
```

Comment:
Does is matter that this function isn't `async`?

Reply:
Well. Alternatives would be:
- inherit and add a method to the `AsyncClient`
- add a `await loop.run_in_executor(None, lambda: fetch_changeset(...))`

What do you think?

Reply:
I think ultimately we should add a method to the `AsyncClient`, but we can do your second suggestion to get this merged as long as we get a follow-on issue.